index.js

```diff
@@ -13,28 +16,60 @@ module.exports = {
       name: 'ember-cli-babel',
       ext: 'js',
       toTree: function(tree) {
-        return require('broccoli-babel-transpiler')(tree, getOptions(addon));
+        return require('broccoli-babel-transpiler')(tree, getBabelOptions(addon));
       }
     };
 
     registry.add('js', plugin);
   },
 
+  shouldIncludePolyfill: function() {
+    var options = getAddonOptions(this);
+    return options.includePolyfill === true;
+  },
+
+  importPolyfill: function(app) {
+    app.import('vendor/browser-polyfill.js');
```

Comment:
Should `{ prepend: true }` be added to this import statement so that the polyfill is included before Ember itself?